src/common/prop.c

```diff
@@ -674,6 +701,15 @@ daos_prop_copy(daos_prop_t *prop_req, daos_prop_t *prop_reply)
 				D_GOTO(out, rc);
 
 			roots_alloc = true;
+		} else if (type == DAOS_PROP_PO_POLICY) {
+			pd = entry_req->dpe_val_ptr;
+			pd_reply = entry_reply->dpe_val_ptr;
+			D_ALLOC(pd, sizeof(*pd_reply));
```

Comment:
D_ALLOC_PTR() would be better here as it gets the size automatically.  I know pd and pd_reply are the same type but this line looks odd.